### PR TITLE
Added profile in pom.xml for MacOS M1/M2 chip to use native Netty DNS…

### PR DESCRIPTION
**Pull request by rampler**
Implements #4707

### CODE REVIEW

**Comment by vietj** [comment]
can you sign the Eclipse ECA ?

**Review by vietj — CHANGES_REQUESTED** [review]
can you also add support for OSX native KQueue ?

**Comment by rampler** [comment]
Done. Unfortunately I needed to add some maven property to activate arch-based dependency. If you have idea how to do it more elegant, just tell.

File: .github/workflows/ci-4.x.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@ jobs:
             profile: ''
           - os: ubuntu-latest
             jdk: 8
-            profile: '-PtestNativeTransport'
+            profile: '-DaddMacSupport -PtestNativeTransport'
```
Comment (vietj) [inline .github/workflows/ci-4.x.yml:15]: I don't think we need this as we only test on Linux and Windows for now

**Comment by vietj** [comment]
why is the property necessary ? I don't get that

File: .github/workflows/ci-4.x.yml (hunk 1 above)
Comment (rampler) [reply .github/workflows/ci-4.x.yml]: Changes from workflows reverted

**Comment by vietj** [comment]
thanks I took inspiration from your PR and made the change myself https://github.com/eclipse-vertx/vert.x/pull/4714

**Pull request closed without merging**